pcs_detection/scripts/data/fcn8_train.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+'''
+ * @file fcn8_train.py
+ * @brief Used for training neural nets, and view the images passed into the network after preprocessing
+ *
+ * @author Jake Janssen
+ * @date November 6, 2019
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2017, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ '''
+
+import json
+import os        
+from src_python.pcs_detection.process import test_dataloader, train
```

Comment:
I realize you are not installing this with cmake, but when you do it will install the pcs_detection module without you having to do this. In fact, I'm a bit surprised this works unless you are doing something tricky with your Python path.

Long story, short please change `src_python.pcs_detection.process` to `pcs_detection.process`

Reply:
Ah that makes sense, I could not get the import from pcs_detection.process to work so I ran it from the outer pcs_detection directory